CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [[v8.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v8.2.0...HEAD)] - 2020-xx-xx]
 
+- Added instructions how to build cluster with Linux/Windows nodes (by @ivanguravel)
```

Comment:
I think we need to keep this as simple possible. So let's exclude "linux" here because that's covered in all other examples.

```suggestion
- Added instructions for how to add Windows nodes (by @ivanguravel)
```